src/resources/Microsoft.Authorization/policyDefinitions/Deny-VMUnmanagedDisk.json

```diff
@@ -0,0 +1,67 @@
+{
+    "name": "Deny-VM-with-unmanaged-disk",
+    "apiVersion": "2021-06-01",
+    "scope": "null",
+    "properties": {
+        "policyType": "Custom",
+        "mode": "all",
+        "displayName": "Deny virtual machines and virtual machine scalesets not using managed disk",
+        "description": "Deny virtual machines not using managed disk. It checks the managedDisk property on virtual machine OS, and Datadisk fields.",
+        "metadata": {
+            "version": "1.0.0.0",
+            "category": "Compute",
+            "source": "https://github.com/Azure/Enterprise-Scale/",
+            "alzCloudEnvironments": [
+                "AzureCloud"
+            ]
```

Comment:
Should the other clouds be supported here as we know managed and unmanaged disk are available in china and us gov?

Reply:
If we are sure, ok. Added the others back in.